src/main/java/emily/command/Emily.java

```diff
@@ -0,0 +1,68 @@
+package main.java.emily.command;
+
+import java.io.File;
+import java.util.Scanner;
+
+/**
+ * Main program
+ */
+public class Emily {
+
+    private static final String FILE_PATH = "data/emily.txt";
+    private final Storage storage;
+    private TaskList tasks;
+    private final Ui ui;
+
+    /**
+     * Initialises Emily bot
+     * @param filePath location to store the information
```

Comment:
I think there should be an empty line between the description and parameter sections. I noticed the same issue in a few other places as well.